components/ui/tabcounter/src/main/AndroidManifest.xml

```diff
@@ -2,4 +2,6 @@
    - License, v. 2.0. If a copy of the MPL was not distributed with this
    - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="mozilla.components.ui.tabcounter" />
+    package="mozilla.components.ui.tabcounter" >
+    <application android:supportsRtl="true" />
```

Comment:
I think this will require another issue.  Adding RTL support makes sense but this requires a lot of testing to confirm that we do support it correctly in `tabcounter`

Reply:
Hmm, i just added it for detekt.

Reply:
how should it be because it was wanted by detekt. Need to convert it to false?

Reply:
Completely removed.